src/main/holochainManager.ts

```diff
@@ -430,6 +455,9 @@ export class HolochainManager {
     );
 
     await this.adminWebsocket.enableApp({ installed_app_id: appId });
+    if (icon) {
```

Comment:
This is redundant, it already gets stored as part of `storeUiIfNecessary`

Reply:
Hmm...there is an edge case though where you have installed the same app from filesystem already and now install another instance from appstore. In that case if we remove this step, there would be no icon stored...

Reply:
If we keep it, the app that you installed from filesystem and had no icon suddenly has one which is also a bit confusing.